### DISCUSSION TITLE
Having Docker images in a local registry, but getting new versions from Docker Hub

### DISCUSSION THREAD

**Discussion by interN3rd** [renovatebot/renovate#21185] [Request Help]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

35.x

### If you're self-hosting Renovate, select which platform you are using.

Bitbucket Server

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

In the end, I want Renovate to scan the local Docker registry of my organization for current Docker images in use and I want Renovate to lookup new versions available in Docker Hub for it to then create a pull request.

Let's say, I have a Docker image in my organization's local repository and it's version `{Some_Docker_Image}:1.1.1`. I then want Renovate to know about this Docker image and to lookup new versions of `{Some_Docker_Image}:1.1.1` in Docker Hub, to then create a pull request. Let's say, `{Some_Docker_Image}:1.2.0` is available in Docker Hub. I then want to configure Renovate in a way, that it know about this new version from Docker Hub and to create a pull request. This pull request would then be the notification to me, so I know that I have to update the local Docker registry.

Is there any way to configure Renovate that way or am I trying to apply a wrong solution to an otherwise solvable problem?

### What you tried so far.

- Searched the discussions for similiar problems, but didn't find it
- Read through the documentation to find an option for configuration, but wasn't sure whether any of it fits my needs

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment]
Try setting a package rule with matchDatasources=docker and registryUrls=https://your-registry
